libs/build-openssl-desktop.sh

```diff
@@ -92,4 +102,10 @@ cp -p "$OPENSSL_OUTPUT_PATH"/lib/libcrypto.a "$OPENSSL_DIR""/lib"
 cp -L "$PWD"/include/openssl/*.h "${OPENSSL_DIR}/include/openssl"
 rm -rf "$OPENSSL_OUTPUT_PATH"
 
+if [[ "$CROSS_COMPILE_TARGET" =~ "win32-x86-64" ]]; then
+  # See https://www.wagner.pp.ru/~vitus/articles/openssl-mingw.html.
```

Comment:
Hm, I think that page implies we don't need this in current versions of openssl (or maybe it's just the rest of the steps we don't need).

Doesn't really matter either way to me though.

Reply:
Unclear?  Let's try to get to OpenSSL 1.1.x ASAP.